packages/opencensus-web-core/test/test-initial-load-util.ts

```diff
@@ -18,21 +18,22 @@ import {
   isSampled,
   makeRandomSamplingDecision,
   setInitialLoadSpanContext,
+  initialLoadSpanContext,
```

Comment:
Rather than exporting the variable, could we export `getInitialLoadSpanContext`?